src/electron/custom_install_steps.nsh

```diff
@@ -65,7 +65,7 @@ ${StrNSISToIO}
   StrCmp $0 0 installservice
   MessageBox MB_OK "Sorry, we could not configure your system to connect to Outline. Please try \
     running the installer again. If you still cannot install Outline, please get in \
-    touch with us and let us know that the TAP device failed to install with this error:$\n$\n$1"

```

Comment:
We could consider leaving this in, given that some users have resorted to sending screenshots.

Reply:
Yes, but with the device names now included the output is very long - since we receive the output anyway, I think it's probably better not to encourage that.